### PR TITLE
Update app Protocol Version to 70027

### PR DESCRIPTION
**Pull request by blondfrogs**

### CODE REVIEW

**Comment by akshaynexus** [comment]
ACK

File: ravenwallet/Info.plist [hunk 1]
```diff
@@ -85,7 +85,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>5</string>
+	<string>$(CURRENT_PROJECT_VERSION)</string>
```
Comment (cfox) [inline ravenwallet/Info.plist:88]: Is moving this from 5 back to 1 going to cause any problems?

File: Modules/core/BRPeer.c [hunk 2]
```diff
@@ -58,8 +58,8 @@
 #define MAX_MSG_LENGTH     0x02000000
 #define MAX_GETDATA_HASHES 50000
 #define ENABLED_SERVICES   0ULL  // we don't provide full blocks to remote nodes
-#define PROTOCOL_VERSION   70025
-#define MIN_PROTO_VERSION  70017 // peers earlier than this protocol version not supported (need v0.9 txFee relay rules)
+#define PROTOCOL_VERSION   70027
+#define MIN_PROTO_VERSION  70026 // peers earlier than this protocol version not supported (need v0.9 txFee relay rules)
```
Comment (cfox) [inline Modules/core/BRPeer.c:62]: Verified this is the version in the current release: https://github.com/RavenProject/Ravencoin/blob/release_3.3.1/src/version.h#L13

**Review by spyder46n2 — APPROVED** [review]
Looks good.

**Pull request merged**